src/ontology/reports/sync-synonym/doid.synonyms.confirmed.robot.tsv

```diff
@@ -9132,8 +9132,8 @@ MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		craniocarpotarsal dy
 MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		craniocarpotarsal dystrophy			DOID:0111604	Freeman-Sheldon syndrome					craniocarpotarsal dystrophy	DOID:0111604										
 MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		whistling face syndrome			DOID:0111604	Freeman-Sheldon syndrome					whistling face syndrome	DOID:0111604										
 MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		whistling face-windmill vane hand syndrome			DOID:0111604	Freeman-Sheldon syndrome					whistling face-windmill vane hand syndrome	DOID:0111604										
-MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		distal arthrogryposis type 2A			DOID:0111605	distal arthrogryposis type 2A					distal arthrogryposis type 2A	DOID:0111605										
 MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		distal arthrogryposis type 2A	distal arthrogryposis type 2A	distal arthrogryposis type 2a	DOID:0111605	distal arthrogryposis type 2A					distal arthrogryposis type 2A	DOID:0111605										
+MONDO:0008675	Freeman-Sheldon syndrome	oio:hasExactSynonym		distal arthrogryposis type 2A			DOID:0111605	distal arthrogryposis type 2A					distal arthrogryposis type 2A	DOID:0111605										
```

Comment:
## Many (most?) diffs are spurious
This diff is not caused by the changes in this PR (#738). This is an example of a diff that is caused because of duplicate rows (sorting isn't set up to account for that), which results in some rows being shifted up or down a little bit.

Fixed in #720. Reviewing these synonym sync diffs will be a lot easier once that's merged.